src/wms/src/main/java/org/geoserver/wms/WMSInfoImpl.java

```diff
@@ -231,26 +230,26 @@ public void setGetMapMimeTypeCheckingEnabled(boolean getMapMimeTypeCheckingEnabl
 
     @Override
     public String getRootLayerTitle() {
-        if (rootLayerTitle == null && internationalRootLayerTitle != null)
-            return internationalRootLayerTitle.toString(GeoServerDefaultLocale.get());
         return rootLayerTitle;
     }
 
     @Override
     public void setRootLayerTitle(String rootLayerTitle) {
-        this.rootLayerTitle = rootLayerTitle;
+        if (rootLayerTitle != null) {
```

Comment:
This looks broken, one should be able to set a null title. Same for the abstract below.

Reply:
fixed.